test/specs/validate-spec/invalid/3.x/array-no-items.yaml

```diff
@@ -0,0 +1,16 @@
+openapi: 3.0.0
+info:
+  version: "1.0.0"
+  title: Invalid API
+
+paths:
+  /users:
+    parameters:
+      - name: tags
+        in: query
+        schema:
+          type: array
```

Comment:
We've long had handling for this in https://github.com/readmeio/oas/blob/main/src/lib/openapi-to-json-schema.ts#L413-L415, and this handling was already present for Swagger 2.0 specs in this library, but it'll be nice to root out these quirks at validation time for OAS 3.0+.